### PR TITLE
Feature/category tabs

### PR DESCRIPTION
**Pull request by messkalina**
I created a scrollable horizontal tabs component. The scroll is executed by clicking the arrows buttons. The left arrow button appears only when we scroll to the right. It is not visible when scrolling left is not possible.

**The component is functional however the hover and active states for tabs items only work for the first 5 tabs.** I dont know how to fix it at this point. I will create a ticket to fix this issue later, once the component is merged.

Screenshot 2024-10-17 at 11 32 32
Screenshot 2024-10-17 at 11 32 42

### CODE REVIEW

**Comment by 006080** [comment]
Here also a conflict

**Comment by messkalina** [comment]
> Here also a conflict

The conflict is resolved, ready to merge:)

**Comment by messkalina** [comment]
I resolved latest merge conflict and the PR is ready to be merged. Pls review, thank you!

**Comment by 006080** [comment]
To improve the .category-tabs style, I recommend removing the padding and replacing the border-bottom with a subtle shadow effect for a cleaner look:
.category-tabs {
  display: flex;
  overflow-x: auto;
  white-space: nowrap;
  /* padding: 10px 0; */
  /* border-bottom: 1px solid #ddd; */
  scroll-behavior: smooth;
}
Additionally, since we're standardizing on modular CSS, we should refactor this into a modular component for better reusability and maintenance.

File: src/components/CategoryTabs/CategoryTabs.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+.category-tabs {
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+  padding: 10px 0;
+  border-bottom: 1px solid #ddd;
+  scroll-behavior: smooth;
+}
+
```
Comment (006080) [inline src/components/CategoryTabs/CategoryTabs.css:23]: To improve the .category-tabs style, I recommend removing the padding and replacing the border-bottom with a subtle shadow effect for a cleaner look:
.category-tabs {
  display: flex;
  overflow-x: auto;
  white-space: nowrap;
  /* padding: 10px 0; */
  /* border-bottom: 1px solid #ddd; */
  scroll-behavior: smooth;
}

**Review by 006080 — CHANGES_REQUESTED** [review]
To improve the .category-tabs style, I recommend removing the padding and replacing the border-bottom with a subtle shadow effect for a cleaner look:
.category-tabs {
  display: flex;
  overflow-x: auto;
  white-space: nowrap;
  /* padding: 10px 0; */
  /* border-bottom: 1px solid #ddd; */
  scroll-behavior: smooth;
}

**Comment by 006080** [comment]
Additionally, since we're standardizing on modular CSS, we should refactor this into a modular component for better reusability and maintenance.

**Comment by 006080** [comment]
<div className={styles.categoryTabsContainer}> {/* Modular CSS syntax example */}

**Comment by 006080** [comment]
<div className={styles.categoryTabsContainer}> {/* Modular CSS syntax example */}

**Comment by 006080** [comment]
div className={styles.categoryTabsContainer}     This is example of the sintax

**Comment by messkalina** [comment]
> Hey, sorry. Maybe I didn't get a point but I see that CSS file wasn't converted. Styles will never apply in that way:(
> 
> Modular CSS classes are scoped locally to the component to avoid conflicts. After importing the CSS module (e.g., import styles from './Component.module.css'), each CSS class is accessed as a property on the styles object, using styles.className.
> 
> So, category-tabs-container in standard CSS becomes categoryTabsContainer in the CSS module file (Component.module.css).

Ok, I was googling how to tackle this, I dont know how to convert them. I will ask for help on that then. Thank you!

**Pull request merged**